twine/utils.py

```diff
@@ -201,9 +204,9 @@ def check_status_code(response: requests.Response, verbose: bool) -> None:
     except requests.HTTPError as err:
         if response.text:
             if verbose:
-                print("Content received from server:\n{}".format(response.text))
+                logger.info("Content received from server:\n{}".format(response.text))
             else:
-                print("NOTE: Try --verbose to see response content.")
+                logger.warning("NOTE: Try --verbose to see response content.")
```

Comment:
Good observation re: the `NOTE`. With that in mind, I might write this as:

```python
logger.info(f"Content received from server:\n{response.text}")
if not verbose:
    logger.warning("NOTE: Try --verbose to see response content.")
```

That will decouple the `info` log from the verbosity setting (as it is and will be in other instances), which means it would always respect the global logging level.